managers/parsers/inTechOpenParser.py

```diff
@@ -0,0 +1,60 @@
+from managers.parsers.abstractParser import AbstractParser
+from managers.webpubManifest import WebpubManifest
+
+
+class inTechOpenParser(AbstractParser):
+    ORDER = 6
+
+    def __init__(self, uri, mediaType, record):
+        super().__init__(uri, mediaType, record)
+
+        self.source = self.record.source
+        self.identifier = list(self.record.identifiers[0].split('|'))[0]
+
+    def validateURI(self):
+        return super().validateURI()
+
+    def createLinks(self):
+        s3Root = self.generateS3Root()
+
+        if self.mediaType == 'application/pdf':
```

Comment:
re: your question. The identifier that the website uses seems to only be consistently available as part of the URL we use to construct a manifest and download the book from the `application/pdf` link.

Right now it seems like the best solution would be to extract that identifier from here using a regex and then make sure that it is in the `text/html` URL later on.

The only issue is that this relies on those links being sent in the right order... perhaps that is something we could improve in the `DOABLinkManager` class?